src/deploy.py

```diff
@@ -48,3 +49,10 @@ def deploy(pname, instance_id=None, branch='master'):
 
     bootstrap.create_update(stackname)        
     setdefault('.active-stack', stackname)
+
```

Comment:
as wrote somewhere up in this PR, if we can build a unique minion id without using the build vars, that's great and we can remove this hack